.github/workflows/smoke.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        containers: [1, 2, 3]
+        containers: [1, 2, 3, 4, 5]
```

Comment:
Why this change? From what I see each container still runs for about 7 minutes. There's more overhead starting the machine than running the tests themselves.

Reply:
@katspaugh we will be adding more tests to the suite. Adding more machines will reduce time spent on tests going forward.